test/profiler/test_profiler.py

```diff
@@ -575,7 +574,7 @@ def call_module(x):
         if kineto_available() and not IS_WINDOWS:
             with TemporaryFileName(mode="w+") as fname:
                 p.export_chrome_trace(fname)
-                with io.open(fname, 'r') as f:
+                with open(fname) as f:
```

Comment:
Can you please point me to the rule claiming that `io.open` and `open` are equivalent in python-3.8+?

Reply:
https://beta.ruff.rs/docs/rules/open-alias/

Reply:
Note that it was already an alias to open in 3.7: https://docs.python.org/3.7/library/io.html#io.open so sounds good.

Reply:
This breaks the profiler test
https://github.com/pytorch/pytorch/pull/105434/files#diff-9a4c92735239449647d2b797612d3c368636544fd36265e89be0f0296eddbc49L4

Reply:
@briancoutinho yes, just use `open`